ducc/webhook/registry-webhook.py

```diff
@@ -46,8 +47,10 @@ def publish_message(notifications_file, action, image):
             if (last_line_id % int(args_dic['rotation']) == 0 and last_line_id != 0):
                 new_notifications_file = str(first_line_id)+'-'+str(last_line_id)+notifications_file.split('/')[-1]
                 os.rename(notifications_file, new_notifications_file)
+                open(notifications_file, 'a').close()
                 with open(new_notifications_file, 'a+') as f:
                     f.write('xx|file rotation|xx\n')
+                time.sleep(1)
```

Comment:
This synchronization logic between `registry-webhook.py` and `registry-listener.go` looks quite fragile, especially given the file rotation process.

I think it would be easier having both sides in the same file. Wouldn't it be possible to call the `cvmfs_ducc ...` command line directly from the Python process?